onthequad/onthequad/UserEndPoints.py

```diff
@@ -0,0 +1,89 @@
+import falcon
+import json
+from .UsersDB import UsersDB
+from .DataBaseSetUp import DataBaseSetUp
+authen = DataBaseSetUp.authentication()
+
+class UserEndPoints:
+
+    def on_get(self, req, resp):
+        
+        send = UsersDB.getAllUsers()
+        resp.body = json.dumps(send)
+        resp.status = falcon.HTTP_200
+
+    def on_post(self, req, resp):
+        data = json.loads(req.stream.read())
+    
+        if any(data):
+
+            firstName = data['firstName']
+            lastName = data['lastName']
+            email = data['email']
+            password = data['password']
+
+            user = authen.create_user_with_email_and_password(email, password)
```

Comment:
Personally I think it might be easier to call the create_user_with_email_and_password inside the createUser method inside the database file instead of doing a little bit of the work in both files.

Reply:
I know but it would be better to follow the story point we created. I thought about that too, it would be easier to do that way; I don't have to import authen method etc. But I stuck to the user story on Jira

Reply:
Since everything you need is in master and has been pulled from master, it would be an easy change.